analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AsyncVoidMethod.cs

```diff
@@ -48,7 +48,7 @@ private async void EventHandlerCases_MyEvent(object sender, bool e)
             // Property changed implementation goes here
         }
 
-        private async void ArbreDesClefs_ItemInvoked(TreeView sender, TreeViewItemInvokedEventArgs args) { }    // Noncompliant FP
+        private async void ArbreDesClefs_ItemInvoked(TreeView sender, TreeViewItemInvokedEventArgs args) { }    // Compliant, doesn't inherit from EventArgs, but looks like it
```

Comment:
what if ` private async void ArbreDesClefs_ItemInvoked(TreeView somethingElse, TreeViewItemInvokedEventArgs arguments)` ? is using `sender` and `args` a pattern enforced by MS guidelines?

Reply:
Can you please clarify this question in more detail? I didn't understand it.
I'll update the comment a little, as the actual fix was about the sender args.